### PR TITLE
Story/remove five gb limit

### PR DESCRIPTION
**Pull request by bartelsb**

### CODE REVIEW

File: commands/db/import.go [hunk 1]
```diff
@@ -23,7 +25,13 @@ import (
 )
 
 func CmdImport(databaseName, filePath, mongoCollection, mongoDatabase string, skipBackup bool, id IDb, ip prompts.IPrompts, is services.IServices, ij jobs.IJobs) error {
-	if _, err := os.Stat(filePath); os.IsNotExist(err) {
+	isPod05 := false
+	versionInfo, err := id.RetrievePodApiVersion()
+	if versionInfo.Version < "4.1.0" {
+		isPod05 = true
```
Comment (forana) [inline commands/db/import.go]: Can we rename this to something like `singleUploadMode`? Seems like we'll run into this in the future, somehow - would be nice if the variable was meaningful :)

Comment (bartelsb) [reply commands/db/import.go]: Updated `isPod05` to `singleUploadMode`.

**Comment by forana** [comment]
local testing passed - LGTM

**Comment by forana** [comment]
version bump LGTM

**Pull request merged**